ambry-utils/src/main/java/com.github.ambry.utils/Throttler.java

```diff
@@ -90,7 +90,7 @@ public void maybeThrottle(double observed) throws InterruptedException {
   /**
    * Disable the throttler for good.
    */
-  public void close() {
+  public void disable() {
```

Comment:
don't we have any tests for close? I don't see any changes in unit tests for Throttler.